Added a news webpage

Kashishkhan08 opened the pull request.

avinash201199 commented:
how these pages is related to this repo ?

avinash201199 commented:
please make valid pr , not just random things in any repo , it won't help anybody

Kashishkhan08 commented:
I'm sorry if you found this irrelevant.
I added this in web folder because I thought we can add our programs. I made this using HTML and CSS so I added in it.
